precommit/src/main/shell/test-patch-docker/Dockerfile

```diff
@@ -149,16 +149,22 @@ FROM yetusbase AS yetushadolint
 ARG HADOLINT_VERSION=2.12.0
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN if [[ "$(uname -m)" == "x86_64" ]]; then curl -sSL \
-        https://github.com/hadolint/hadolint/releases/download/v$HADOLINT_VERSION/hadolint-Linux-"$(uname -m)" \
+        https://github.com/hadolint/hadolint/releases/download/v$HADOLINT_VERSION/hadolint-Linux-x86_64 \
```

Comment:
I still prefer the style I proposed in my patch because docker can cache the downloaded artifacts between runs. The layer caching speeds up rebuild cycles drastically.